<a href="https://colab.research.google.com/github/urviparikh/DATA-690-WANG/blob/main/world_development_explorer/Project_WorldDev.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **Awareness on the Impacts of Climate Change**

Climate change is a phenomenon that is affecting everyone globally because we all essentially share the same environment.



![image](https://user-images.githubusercontent.com/78182982/112765763-a8a69a80-8fdc-11eb-9082-1f5aff5c7070.png)

According to an article on the Washington Post, the current pace of human-caused carbon emissions is likely to trigger irreversible damage to the planet, according to an international study released in July 2020. Scientists believe that human activities--such as burning oil, gas and coal along with deforestation--can push carbon dioxide levels to such levels that the Earth's global temperature can increase between 4 to 8 degrees Fahrenheit. With rising temperatures, there are impacts on drastic weather patterns, snow and ice storms, heat waves and a chain reaction of many other changes affecting the planet. Globally, the year 2020 was the hottest year on record. Climate change has specifically been in the spotlight for the year 2020 also as the U.S. set new records for the number of wildfires in the west coast of the country. Five of California's 10 largest wildfires on record happened in the year 2020 and it has devastated over 4.2 million acres. Not to mention, the amount of Arctic sea ice has declined so rapidly that the region may see ice-free summers by the 2030s, according an article on the New York Times.

So, it becomes extremely important that everyone is aware of the impacts of climate change. Many developing countries are suspected to be major contributors of climate change due to their high emissions of Greenhouse gases. Several countries' growing GDP Per Capita is also believed to be producing higher CO2 emissions, causing further damages to the ever-rising temperatures.

## Research Questions:

**1. Are the developing countries causing more of an impact on climate change compared to the developed countries?**



**2. Is there a link between education and climate change? If a country's citizens are more educated, does that mean they are also more aware of the drastic impacts of climate change?**

**3. Are more countries turning towards renewable energy sources in order to reduce the adverse effects on climate change?**

I was interested in exploring the impacts of wealth, renewable energy and education on the growing concerns of climate change. Utilizing the various indicators present on the WDX website, I was able to obtain data on these research questions. The only available data span from the years 2010 to 2016.

## Indicators to be used:

**1. GDP Per Capita:** This indicator provides per capita values for gross domestic product (GDP) expressed in current international dollars converted by purchasing power parity (PPP) conversion factor. GDP is the sum of gross value added by all resident producers in the country plus any product taxes and minus any subsidies not included in the value of the products. conversion factor is a spatial price deflator and currency converter that controls for price level differences between countries. Total population is a mid-year population based on the de facto definition of population, which counts all residents regardless of legal status or citizenship.

**2. CO2 emissions:** Carbon dioxide emissions are those stemming from the burning of fossil fuels and the manufacture of cement. They include carbon dioxide produced during consumption of solid, liquid, and gas fuels and gas flaring.



**3. Electricity production from renewable sources:** Electricity production from renewable sources, excluding hydroelectric, includes geothermal, solar, tides, wind, biomass, and biofuels.



**4. Barro-Lee: Percentage of population age 15+ with no education:** Percentage of population age 15+ with no education



# Countries to be analyzed:

1. United States
2. Canada
3. India
4. China
5. Australia

These particular countries were chosen because they not only offer a regional comparison among the U.S.-Canada and China-India, but also differ on the developed vs. developing country scale.



In [1]:
import pandas as pd
import plotly.express as px


In [2]:
##create a URL for the indicators
df = pd.read_csv("https://github.com/urviparikh/DATA-690-WANG/raw/main/world_development_explorer/wdi_data.csv")
df.head(5)

Unnamed: 0  Year  ...  Income Group    Lending Type
0           0  2010  ...   High income  Not classified
1           1  2011  ...   High income  Not classified
2           2  2012  ...   High income  Not classified
3           3  2013  ...   High income  Not classified
4           4  2014  ...   High income  Not classified

[5 rows x 10 columns]

In [3]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 30 entries, 0 to 29
Data columns (total 10 columns):
 #   Column             Non-Null Count  Dtype  
---  ------             --------------  -----  
 0   Unnamed: 0         30 non-null     int64  
 1   Year               30 non-null     int64  
 2   EG.ELC.RNWX.KH     30 non-null     int64  
 3   EN.ATM.CO2E.KT     30 non-null     float64
 4   NY.GDP.PCAP.PP.CD  30 non-null     float64
 5   Country Code       30 non-null     object 
 6   Country Name       30 non-null     object 
 7   Region             30 non-null     object 
 8   Income Group       30 non-null     object 
 9   Lending Type       30 non-null     object 
dtypes: float64(2), int64(3), object(5)
memory usage: 2.5+ KB


In [4]:
# The unnamed column will be dropped as a data cleansing process
df.drop(columns=["Unnamed: 0"], inplace=True)
df.head(5)

Year  EG.ELC.RNWX.KH  ...  Income Group    Lending Type
0  2010      8256000000  ...   High income  Not classified
1  2011      9718000000  ...   High income  Not classified
2  2012     12573000000  ...   High income  Not classified
3  2013     14939000000  ...   High income  Not classified
4  2014     18622000000  ...   High income  Not classified

[5 rows x 9 columns]

We will now start producing some figures using this data in order to answer the research questions.

In [5]:
# Impact of GDP Per Capita on their CO2 emissions 
fig = px.scatter(df, x='NY.GDP.PCAP.PP.CD', y='EN.ATM.CO2E.KT', 
                 animation_frame= "Year",
                  color="Country Name",
                 labels={
                        "Year":"Year",
                        "NY.GDP.PCAP.PP.CD": "GDP Per Capita, PPP (current international $)",
                        "EN.ATM.CO2E.KT" : "CO2 emissions (kt)"
                 },
                 title= "CO2 Emissions vs. GDP Per Capita PPP"

)
fig.show()

Although the GDP per capita are not as different for U.S. and Canada, both of which are developed countries of North America, there is a significant rise of CO2 emissions from the U.S.(5.39M) compared to Canada(527k). Although the total population also plays a part in the overall CO2 emitted by a country, the U.S. should be aware of its impact on climate change. Though, it's also interesting to note that the population of China and India are relatively close, but China's CO2 levels are extraordinarily higher at close to 10M vs. India's levels at 2M.

In [6]:
# Line graphs track the progress over the years
fig = px.line (df,
               x="Year", y="NY.GDP.PCAP.PP.CD",
               color="Country Name",
               height=600,
               labels={
                      "Year":"Year",
                       "NY.GDP.PCAP.PP.CD":"GDP Per Capita, PPP"
               },
               title="GDP Per Capita PPP Over the Years"
               
               )
fig.show()

In [7]:
# Line graphs track the CO2 emissions over the years
fig = px.line (df,
               x="Year", y="EN.ATM.CO2E.KT",
               color="Country Name",
               height=600,
               labels={
                      "Year":"Year",
                       "EN.ATM.CO2E.KT":"CO2 emissions (kt)"
               },
               title="Amount of CO2 Emissions Over the Years"
               
               )
fig.show()

These line graphs are able to track the trends of their GDP Per Capita as well as their CO2 emissions from 2010 to 2016.

**Key Takeaways from these graphs:**


As it can be seen, India and China (developing countries)
have had a rather rapid increase in CO2 emssions vs. Canada and Australia (developed countries).




It's staggering to note that both Canada and Austrlia have managed to truly control their CO2 emissions, as they have only risen slightly over the timespan of 6 years.



There is also no correlation between how rich a country is and how much CO2 is emitted. Even though the U.S., Australia and Canada all have comparatively higher GDP Per Capita, it does not seem to have an impact on their CO2 levels.

In [8]:
# Line graph tracks progress of renewable energy sources
fig = px.line (df,
               x="Year", y="EG.ELC.RNWX.KH",
               color="Country Name",
               height=600,
               labels={
                      "Year":"Year",
                       "EG.ELC.RNWX.KH":"Electricity Production from renewable sources, excluding hydroelectric (kWh)"
               },
               title="Electricity Production Over the Years"
               
               )
fig.show()

This line graph tracks the progress on the amount of electricity being produced from renewable sources vs. the non-renewable sources that would increase their CO2 emissions. As it can be seen in the graph, a developing country like China has truly increased their utilization of renewable energy quite drastically, and is very close to the U.S.(developed country) numbers. Similarly, there is a common trend of increasing renewable energy among all of the countries, which can be attributed to increasing educational awareness on climate change.

In [9]:
##creating a URL to determine the relationship with another indicator
df2 = pd.read_csv("https://github.com/urviparikh/DATA-690-WANG/raw/main/world_development_explorer/education_data.csv")
df2.head(5)

Unnamed: 0  Year  ...         Income Group    Lending Type
0           0  2010  ...          High income  Not classified
1           1  2010  ...          High income  Not classified
2           2  2010  ...  Upper middle income            IBRD
3           3  2010  ...  Lower middle income            IBRD
4           4  2010  ...          High income  Not classified

[5 rows x 10 columns]

In [10]:
# The unnamed column will be dropped as a data cleansing process
df2.drop(columns=["Unnamed: 0"], inplace=True)
df2.head(5)

Year  BAR.NOED.15UP.ZS  ...         Income Group    Lending Type
0  2010              0.72  ...          High income  Not classified
1  2010              0.67  ...          High income  Not classified
2  2010              5.35  ...  Upper middle income            IBRD
3  2010             33.16  ...  Lower middle income            IBRD
4  2010              0.35  ...          High income  Not classified

[5 rows x 9 columns]

In [12]:
fig = px.bar(df2,
             x="Country Name",
             y="BAR.NOED.15UP.ZS",
             color="Country Name",
             height=600,
             labels={
                      "Country Name": "Country Name",
                      "sepal_width":"Sepal Width (cm)",
                      "BAR.NOED.15UP.ZS":"Barro-Lee:Percentage of population age 15+ with no education"
             },
             title="Percentage of Uneducated Youth in each country"
             )
fig.show()

**Although there is no data to show the progress of youth education over the years, these numbers from 2010 can provide sufficient information to draw certain conclusions.**



This bar graph clearly defines that there is a link between education and climate change. Two of the main contributors of climate change (India and China) also have the highest percentage of their youth uneducated.


For India, 33% of its population age 15+ has received no education and that significantly impacts their citizens decisions on a day-to-day basis. Youth constitutes the majority of the population in many countries and have an increasingly strong social and environmental awareness, which has the power to transform our societies towards a low-carbon and climate resilient future.

## **From this data and research, it can be concluded that:**


- There is no direct link between wealth and CO2 emissions. The initial hypothesis of growing GDP Per Capita produces higher CO2 emissions cannot be proven with the current data.



- There is definitely a correlation between education and climate change. The more educated a country's youth is, the better they have managed climate change.


- Although there are many contributing factors for climate change, individual countries have to shift their focus independently to truly obtain best results for themselves. Whether it is a developing country or a developed country, it is their responsibility to redcue their carbon footprint and preserve our climate.

## **Future Directions:**


With as many petitions and initiatives taking place all over the globe, we can only hope that there is a downward trajectory for the CO2 emissions in the upcoming years. Specifically, for the top CO2 emitters, China and India, there are efforts being made to significantly reduce their Greenhouse gas emissions. According to an article on the Washington Post, as of June 2020, there were about 1.75 million electric rickshaws roaming on the streets in India and that is higher than the number of electric cars being sold in the U.S.


This is also one of the reasons why the Biden administration has been petitioning for cutting down our Greenhouse Gas emissions. The administration noted that the U.S. will set a new emissions-reduction target before the Leaders' Climate Summit in April 2021. A 50% reduction by 2030 is attainable and will provide major economic and social benefit for the U.S.


Global awareness of climate change is increasing and should continue to increase in order for us to observe a positive trajectory.



### **References:**

https://www.washingtonpost.com/climate-solutions/2020/06/12/india-emissions-climate/?utm_source=google&utm_medium=acq-nat&utm_campaign=eng-hub-search&utm_content=hub-climate_soultions&utm_keyword=&campaignid=8202785432&adgroupid=111150677210&adid=504675287503&gclid=EAIaIQobChMImKC3wtvT7wIVSuDICh2j1gd9EAAYASAAEgKwJPD_BwE

https://www.washingtonpost.com/weather/2020/07/22/climate-sensitivity-co2/

https://www.nytimes.com/2019/12/04/climate/climate-change-acceleration.html

Data obtained from : http://www.worlddev.xyz/